macro/run_simScript.py

```diff
@@ -596,10 +593,11 @@
  gProcessTable = ROOT.G4ProcessTable.GetProcessTable()
  procmu = gProcessTable.FindProcess(ROOT.G4String('muIoni'),ROOT.G4String('mu+'))
  procmu.SetVerboseLevel(2)
-if inactivateMuonNuclearProcess:
-  ROOT.gROOT.ProcessLine('#include "Geant4/G4ProcessTable.hh"')
-  mygMC = ROOT.TGeant4.GetMC()
-  mygMC.ProcessGeantCommand("/process/inactivate muonNuclear")
+
+if simEngine == "muonDIS":
+    ROOT.gROOT.ProcessLine('#include "Geant4/G4ProcessTable.hh"')
+    mygMC = ROOT.TGeant4.GetMC()
+    mygMC.ProcessGeantCommand("/process/inactivate muonNuclear")
```

Comment:
Now that we don't include the original muon anymore, is this necessary? It should be fairly negligible that the outgoing muon undergoes DIS again and in the few cases where it does, I don't think it's an issue.

Reply:
That is true, will remove this. maybe also the inactivateMuonProcesses, I dont find it being used anywhere anymore.